test/controllers/students_controller_test.rb

```diff
@@ -2,8 +2,23 @@
 
 require 'test_helper'
 
-class StudentsControllerTest < ActionDispatch::IntegrationTest
-  # test "the truth" do
-  #   assert true
-  # end
+class StudentsControllerTest < ActionController::TestCase
+  include Devise::Test::ControllerHelpers
+  def setup
+    @request.env['devise.mapping'] = Devise.mappings[:user]
+    sign_in FactoryBot.create(:user)
+  end
+
+  test 'should get index' do
+    get :index
+    assert_response :success
+    assert_not_nil assigns(:students)
+  end
+
+  test 'should show student' do
+    student = FactoryBot.create(:student)
+    get :show, params: { id: student.id }
+    assert_response :success
+    assert_not_nil assigns(:student)
```

Comment:
You can get more specific by saying
```suggestion
    assert_equal assigns(:student), student
```